utils/utils.cpp

```diff
@@ -0,0 +1,57 @@
+#include "utils.h"
+
+#include <glm/glm.hpp>
+#include <sstream>
```

Comment:
No CPP streams, please!

Reply:
sorry, did use it for the first draft but is not used anymore anyway :D